Formula/tnftpd.rb

```diff
@@ -0,0 +1,63 @@
+class Tnftpd < Formula
+  desc "NetBSD's FTP server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/lukemftpd/lukemftpd-51.tar.gz"
+  sha256 "969b8a35fabcc82759da2433973b9606b7b62f73527e76ac8f18d0a19f473c2a"
+
+  depends_on :xcode => :build
+
+  def install
+    system "tar zxvf tnftpd-20100324.tar.gz"
+    Dir.chdir "tnftpd-20100324"
+
+    system "./configure"
+    system "make"
+
+    File.rename "src/tnftpd", "src/ftpd"
+    File.rename "src/tnftpd.man", "src/ftpd.8"
+    File.rename "src/ftpusers.man", "src/ftpusers.5"
+    File.rename "src/ftpd.conf.man", "src/ftpd.conf.5"
+
+    bin.install "src/ftpd"
+    man8.install "src/ftpd.8"
+    man5.install "src/ftpusers.5"
+    man5.install "src/ftpd.conf.5"
+    etc.install "examples/ftpd.conf"
+    etc.install "examples/ftpusers"
+  end
+
+  def caveats
```

Comment:
IMO at best only caveat 2 is desirable here.

Reply:
@DomT4 Ok